boards/shields/semtech_sx1272mb2das/semtech_sx1272mb2das.overlay

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Lemonbeat GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		lora0 = &lora;
+	};
+};
+
+&arduino_spi {
+	status = "okay";
+
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>;	/* D10 */
+
+	lora: sx1272@0 {
+		compatible = "semtech,sx1272";
+		reg = <0x0>;
+		label = "sx1272";
+		spi-max-frequency = <3000000>;
+
+		reset-gpios = <&arduino_header 0 (GPIO_PULL_UP | GPIO_ACTIVE_HIGH)>;   /* A0 */
```

Comment:
Is that pull-up really needed? Together with active-high output it would mean that the default is to constantly reset device.

Reply:
You are right, the pull-up is not needed. I copied it from https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/96b_wistrio/96b_wistrio.dts#L74
But since the reset pin is never configured as an input, it does not make sense here. I will remove this.

Reply:
done